pycontw/pycon.tw
# Update/2023 cfp

## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies


## Description
**Describe what the change is**
* [feat(proposal): remove talk_language from EventInfo and update lang_choices](https://github.com/pycontw/pycon.tw/commit/ae5b3ae11c53bc2c111cc449827f8cf847b95d77)
* [feat(variables): update modal-content-bg and close-color](https://github.com/pycontw/pycon.tw/commit/22b74a310a8164ceeba3890c98cbb4a7f175c1ae)

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Create and edit proposal
2. Check language choices


## Expected behavior
![image](https://user-images.githubusercontent.com/18432820/217600761-2fd5ede8-d454-4d59-95b4-533e46807d04.png)
![image](https://user-images.githubusercontent.com/18432820/217601122-5511c5e1-92a7-49ae-96de-16ef4a7a5808.png)
